app/models/pairing.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 class Pairing < ApplicationRecord
+  SIDE_PLAYER1_CORP = 1
```

Comment:
you can use Pairing.sides as a class method to get a map of the enum values instead of defining new values, but you have to invert it like Pairing.sides.invert to have it keyed by value instead of name.  You can do that to save duplicating the enum here.

We could do well to tidy up enum usage across Cobra but that is a task for another PR.